vm/src/vm/runners/builtin_runner/bitwise.rs

```diff
@@ -166,8 +166,7 @@ impl BitwiseBuiltinRunner {
             self.stop_ptr = Some(stop_ptr);
             Ok(stop_pointer_addr)
         } else {
-            let stop_ptr = self.base;
-            self.stop_ptr = Some(stop_ptr);
+            self.stop_ptr = Some(0);
```

Comment:
Why does this change?

Reply:
It's a bugfix, I can link you to a discussion about it